test/services/SortStatementsServiceSpec.scala

```diff
@@ -0,0 +1,122 @@
+
+    val requestedSecurityStatements = List(SecurityStatementsByPeriod(LocalDate.of(2017, 11, 28),LocalDate.of(2018,2,2),
+      List(SecurityStatementFile("statementfile_00", "download_url_00", 99, SecurityStatementFileMetadata(2017, 11, 28, 2018, 2, 2, Pdf, SecurityStatement, "12345678", 500, "0000000", someRequestId)))))
+
+    val c79Certificates = VatCertificateFile("statementfile_00", "download_url_00", 99L,
+      VatCertificateFileMetadata(2017, 12, Pdf, C79Certificate,None))
+    val c79Certificates_2 = VatCertificateFile("statementfile_00", "download_url_00", 99L,
+      VatCertificateFileMetadata(2017, 11, Pdf, C79Certificate,None))
+    val c79Certificates_3 = VatCertificateFile("statementfile_00", "download_url_00", 99L,
+      VatCertificateFileMetadata(2017, 12, Pdf, C79Certificate,someRequestId))
+    val c79Certificates_4 = VatCertificateFile("statementfile_00", "download_url_00", 99L,
+      VatCertificateFileMetadata(2017, 11, Pdf, C79Certificate,someRequestId))
+    val c79CertificateFiles = Seq(c79Certificates, c79Certificates_2, c79Certificates_3, c79Certificates_4)
+
+    val requestedC79Certificates = List(VatCertificatesByMonth(LocalDate.of(2017,12,1),
+      List(VatCertificateFile("statementfile_00", "download_url_00", 99,
+        VatCertificateFileMetadata(2017, 12, Pdf, C79Certificate, Some("Ab1234")), ""))),
+      VatCertificatesByMonth(LocalDate.of(2017,11,1),
+        List(VatCertificateFile("statementfile_00", "download_url_00", 99,
+          VatCertificateFileMetadata(2017, 11, Pdf, C79Certificate, Some("Ab1234")), ""))))
+
+    val postponedVatStatement = PostponedVatStatementFile("statementfile_00", "download_url_00", 99L,
+      PostponedVatStatementFileMetadata(2017, 12, Pdf, PostponedVATStatement, "CDS", None))
+    val postponedVatStatement_2 = PostponedVatStatementFile("statementfile_00", "download_url_00", 99L,
+      PostponedVatStatementFileMetadata(2017, 11, Pdf, PostponedVATStatement, "Chief", someRequestId))
+    val postponedVatStatement_3 = PostponedVatStatementFile("statementfile_00", "download_url_00", 99L,
+      PostponedVatStatementFileMetadata(2017, 12, Pdf, PostponedVATStatement, "CDS", someRequestId))
+    val postponedVatStatementFiles = Seq(postponedVatStatement, postponedVatStatement_2, postponedVatStatement_3)
+
+    val requestedPVATStatements = List(PostponedVatStatementsByMonth(LocalDate.of(2017,12,1),
+      List(PostponedVatStatementFile("statementfile_00", "download_url_00", 99,
+        PostponedVatStatementFileMetadata(2017, 12, Pdf, PostponedVATStatement, "CDS", Some("Ab1234")), ""))),
+      PostponedVatStatementsByMonth(LocalDate.of(2017,11,1),
+        List(PostponedVatStatementFile("statementfile_00", "download_url_00", 99,
+          PostponedVatStatementFileMetadata(2017, 11, Pdf, PostponedVATStatement, "Chief", Some("Ab1234")), ""))))
+
+    val dutyDeferementFile: DutyDefermentStatementFile = DutyDefermentStatementFile("2018_03_01-08.pdf", "url.pdf", 1024L,
```

Comment:
Better to give a line break before line number 105, 107, 109 and 111. I know these are in the group but cause these are massive objects so adding a line break make these readable